dist/bin/dotc

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
+  ## resolve links - $0 may be a link to PROG_HOME
+  PRG="$0"
+
+  # need this for relative symlinks
+  while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+      PRG="$link"
+    else
+      PRG="`dirname "$PRG"`/$link"
+    fi
+  done
+
+  saveddir=`pwd`
+
+  PROG_HOME=`dirname "$PRG"`/..
+
+  # make it fully qualified
+  PROG_HOME=`cd "$PROG_HOME" && pwd`
+
+  cd "$saveddir"
+fi
+
+source "$PROG_HOME/bin/common"
+
+default_java_opts="-Xmx768m -Xms768m"
+bootcp=true
+
+CompilerMain=dotty.tools.dotc.Main
+FromTasty=dotty.tools.dotc.FromTasty
+ReplMain=dotty.tools.dotc.repl.Main
+
+PROG_NAME=$CompilerMain
+
+addJava () {
+  java_args+=("$1")
+}
+addScala () {
+  scala_args+=("$1")
+}
+addResidual () {
+  residual_args+=("$1")
+}
+
+classpathArgs () {
+  # echo "dotty-compiler: $DOTTY_COMP"
+  # echo "dotty-interface: $DOTTY_INTF"
+  # echo "dotty-library: $DOTTY_LIB"
+  # echo "scala-asm: $SCALA_ASM"
+  # echo "scala-lib: $SCALA_LIB"
+  # echo "scala-xml: $SCALA_XML"
+  # echo "sbt-intface: $SBT_INTF"
+
+  toolchain=""
+  toolchain+="$SCALA_LIB$PSEP"
+  toolchain+="$DOTTY_LIB$PSEP"
+  toolchain+="$SCALA_ASM$PSEP"
+  toolchain+="$SBT_INTF$PSEP"
+  toolchain+="$DOTTY_INTF$PSEP"
+  toolchain+="$DOTTY_LIB$PSEP"
+  toolchain+="$DOTTY_COMP"
+
+  if [[ -n "$bootcp" ]]; then
+    jvm_cp_args="-Xbootclasspath/a:$toolchain"
+  else
+    jvm_cp_args="-classpath $toolchain"
+  fi
+}
+
+while [[ $# -gt 0 ]]; do
+case "$1" in
+           --) shift; for arg; do addResidual "$arg"; done; set -- ;;
+     -h|-help) help=true && shift ;;
+  -v|-verbose) verbose=true && shift ;;
+    -debug) debug=true && shift ;;
+    -q|-quiet) quiet=true && shift ;;
+
+    # Optimize for short-running applications, see https://github.com/lampepfl/dotty/issues/222
+    -Oshort) addJava "-XX:+TieredCompilation -XX:TieredStopAtLevel=1" && shift ;;
+        -repl) PROG_NAME="$ReplMain" && shift ;;
+       -tasty) PROG_NAME="$FromTasty" && shift ;;
+     -compile) PROG_NAME="$CompilerMain" && shift ;;
+         -run) PROG_NAME="$ReplMain" && shift ;;
+      -bootcp) bootcp=true && shift ;;
+   -nobootcp) unset bootcp && shift ;;
+      -colors) colors=true && shift ;;
+   -no-colors) unset colors && shift ;;
+      -jrebel) jrebel=true && shift ;;
+   -no-jrebel) unset jrebel && shift ;;
```

Comment:
@felixmulder I get them from the `bin/dotc` and `bin/dotr`. It seems it's fine to remove them. Doing it now.